dynawo/sources/Models/Modelica/Dynawo/Electrical/HVDC/HvdcVSC/BaseControls/ACVoltageControl/package.order

```diff
@@ -1,3 +1,4 @@
 QRefQU
 QRefLim
 ACVoltageControl
+QCombCalcul
```

Comment:
Could be named QMinMaxCombCalculation

Reply:
Also, you need to add it to the CMakeLists.txt

Reply:
ok